src/management-system-v2/lib/process-export/export-preparation.ts

```diff
@@ -97,6 +101,22 @@ function getImagesReferencedByHtml(html: string) {
   }
 }
 
+/**
+ * Returns the ids of all subprocesses in the given bpmn that are not expanded
+ *
+ * @param bpmn
+ */
+async function getCollapsedSubprocessIds(bpmn: string) {
```

Comment:
I think renaming this function might be useful because this function is not only returning Ids of collapsed subprocesses but also names if existing.